llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -1171,13 +1172,15 @@ AArch64TargetLowering::AArch64TargetLowering(const TargetMachine &TM,
   if (Subtarget->isNeonAvailable()) {
     // FIXME: v1f64 shouldn't be legal if we can avoid it, because it leads to
     // silliness like this:
+    // clang-format off
     for (auto Op :
          {ISD::SELECT,         ISD::SELECT_CC,
           ISD::BR_CC,          ISD::FADD,           ISD::FSUB,
           ISD::FMUL,           ISD::FDIV,           ISD::FMA,
           ISD::FNEG,           ISD::FABS,           ISD::FCEIL,
           ISD::FSQRT,          ISD::FFLOOR,         ISD::FNEARBYINT,
-          ISD::FSIN,           ISD::FCOS,           ISD::FPOW,
+          ISD::FSIN,           ISD::FCOS,           ISD::FTAN,
+          ISD::FPOW,
```

Comment:
Can you rejig this so there isn't a single opcode value on a line?

Reply:
That would cascading line changes, which I thought might make the change look bigger than it is. . That said I could but I was hoping to create some room for the other trig intrinsics I have planned to add from that RFC. No strong feeling though, so if youwant it i'll make it happen.

Reply:
I think I would rejig them again later if needed. It's a shame clang-format does such an ugly job with it. It's nice to leave it clean in the meantime though.

Reply:
done